scripts/reactive_runner.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright 2024 Canonical Ltd.
```

Comment:
@yanksyoon  Can you elaborate on the second part of your comment? You mean having `reactive.runner.reactive_runner` in a script?

Reply:
In `openstack_manager.py` module, the `def _create_runner` [method](https://github.com/canonical/github-runner-operator/blob/main/src/openstack_cloud/openstack_manager.py#L613) is defined as a static method. I was wondering if this would also help if it was put here as a script as well :) (as to match the style in which we spawn sub processes)

Reply:
We can discuss this further internally. I am resolving this conversation now.